kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -309,76 +334,96 @@ define([
          */
         renderFiles: function (files) {
             const stagingAreaViewer = this;
+            const option = html.tag('option'),
+                button = html.tag('button'),
+                span = html.tag('span'),
+                select = html.tag('select'),
+                div = html.tag('div'),
+                optgroup = html.tag('optgroup'),
+                input = html.tag('input');
             files = files || [];
-            const emptyMsg = 'No files found.';
-
-            const $fileTable = $(
-                stagingAreaViewer.ftpFileTableTmpl({
-                    files: files,
-                    uploaders: stagingAreaViewer.uploaders.dropdown_order,
-                })
-            );
-
-            stagingAreaViewer.$elem.append($fileTable);
+            const emptyMsg = 'No files found.',
```

Comment:
Modified some text to reflect that we're looking at a file system, not just table entries.